### DISCUSSION TITLE
Preview deployment failing - Error: /lib64/libz.so.1: version `ZLIB_1.2.9' not found

### DISCUSSION THREAD

**Discussion by benslv** [vercel/vercel#6945] [Help]
Hey all, I was hoping I might be able to get a bit of help with this error I've encountered?

![image](https://user-images.githubusercontent.com/21127383/140349786-223aa2cf-dcd5-4a69-b3e0-f0ed38e122b4.png)

I recently started work on adding [Blurhash](https://blurha.sh/) functionality to the images on my site, and create a custom function for it, using the `blurhash` package and `sharp` to resize the images before processing - it was far too slow otherwise.

The only issue is that's it's causing my Preview Deployments on Vercel to fail and I can't work out how to fix it. When building the site locally it all works perfectly and I see no errors whatsoever, but it's different on Vercel for some reason and I'm not sure why.

Let me know what other things would be helpful to post and I can add them to this post or as a new reply. Thanks :)

**Comment by Goduu** [comment]
Hi @benslv I had this issue not long ​ago. 
The solution was to downgrade canvas back to the version 2.6.1 what was not so great, because it worked in production but this version triggered another error when I am running it locally. So I need to work with 2 versions one for production and other one for dev. 
If you find another solution, I'm interested. But meanwhile it could maybe help you.

**Reply by scarstens** [reply]
Better solution at https://github.com/Automattic/node-canvas/issues/1779#issuecomment-895885846
